WooCommerce/Classes/ViewRelated/Editor/EditorFactory.swift

```diff
@@ -11,10 +11,19 @@ final class EditorFactory {
 
     // MARK: - Editor: Instantiation
 
+    func productNameEditor(product: Product,
+                                  onContentSave: @escaping Editor.OnContentSave) -> Editor & UIViewController {
+        let navigationTitle = NSLocalizedString("Title", comment: "The navigation bar title of the Aztec editor screen.")
+        let viewProperties = EditorViewProperties(navigationTitle: navigationTitle, formatBarHidden: true)
+        let editor = AztecEditorViewController(content: product.name, viewProperties: viewProperties)
+        editor.onContentSave = onContentSave
```

Comment:
sorry I didn't mention it before - let's use `TextViewViewController` (a simple view controller with a `UITextView` designed for this use case) instead of Aztec, since Aztec might be heavyweight (e.g. it has two text views, one for Rich Text and the other for source code)?

Reply:
Thanks for the tip @jaclync. I removed Aztec and now used `TextViewViewController`. I updated also the PR description following the changes. 🙇